pontoon/messaging/static/js/messaging.js

```diff
@@ -309,12 +362,19 @@ $(function () {
       success: function () {
         Pontoon.endLoader('Message sent.');
         if (!sendToMyself) {
-          container.find('.left-column .sent a').click();
+          const count = container.find('.left-column .sent .count');
+          // Update count in the menu
+          count.html(parseInt(count.html(), 10) + 1);
```

Comment:
You'll need to store the non-formatted serialization e.g. in a `data-count` field, because this will fail for values over a thousand:
```js
new Intl.NumberFormat('en').format(1000) === '1,000'
parseInt('1,000', 10) === 1
```

Reply:
The number is not formatted, so it'll be:
```js
parseInt('1000', 10) === 1000
```

Reply:
This is a different count? https://github.com/mozilla/pontoon/blob/9ae8f44bd2bde67a8eb04b320a23c36c9c9c85c8/pontoon/messaging/static/js/messaging.js#L91-L96

Reply:
Yeah, this is the recipient count. The one you commented on originally is the sent message count.